core/chains/evm/config/toml/defaults/Berachain_Testnet.toml

```diff
@@ -0,0 +1,18 @@
+ChainID = '80084'
+# finality_depth: instant
+FinalityDepth = 10 
+# block_time: 5s 
+LogPollInterval = '10s' 
```

Comment:
I was told we should bump this values with "some" buffer. 
Just to not getting errors or warns that do not make sense. 

Do you want me to change all configs to have the `LogPollInterval` as the `block_time` from the compat tests?

Reply:
Yea this may not be that critical & not sure if we will have some impact of increasing this value 2x, maybe the OCR reports will be slowed.
@dhaidashenko can you please chime in here?

Reply:
we already used 10s for running the ping pong tests and the soak ones, so to avoid re-running everything again will leave it as it is as there is no real concern about this value (per comments)
Thanks (closing this comment)